apps/src/redux/instructions.js

```diff
@@ -236,6 +240,7 @@ export const substituteInstructionImages = (htmlText, substitutions) => {
  * @param {string} config.level.instructions2
  * @param {string} config.level.markdownInstructions
  * @param {array} config.level.inputOutputTable
+ * @param {array} config.level.videos
```

Comment:
appOptions.js has a bunch of JSDOC that tries to document what appOptions looks like. Would be good to also add an @param tag for videos there.